pkg/controller/testing/table.go

```diff
@@ -265,6 +280,23 @@ func (tc *TestCase) VerifyWantAbsent(c client.Client) error {
 	return nil
 }
 
+// VerifyWantEvent verifies that the eventRecorder does contain the events
+// expected to be emitted after reconciliation.
+func (tc *TestCase) VerifyWantEvent(eventRecorder *MockEventRecorder) error {
+	if !reflect.DeepEqual(tc.WantEvent, eventRecorder.events) {
```

Comment:
Does this mean that the order of the wanted events needs to match the order they were actually seen?

Reply:
That is correct. I'll update that in the comment. Let me know if you prefer something else.

Reply:
Now that I think about it, you are probably right, the ordering of these is important and we shouldn't just compare them as a set.